depth_first_traversal/DepthFirstTraversal.java

```diff
@@ -0,0 +1,38 @@
+class DepthFirstTraversal {
+
+    public static int[] arr;    // DFS traversal path
+    public static int k = 0;
```

Comment:
Detects when a class or interface does not have a package definition.
Example(s):
// no package declaration
public class ClassInDefaultPackage {
}